cobalt/browser/application.cc

```diff
@@ -212,21 +212,31 @@ GURL GetInitialURL(bool should_preload) {
   GURL initial_url = GURL(kDefaultURL);
   // Allow the user to override the default URL via a command line parameter.
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
-  if (command_line->HasSwitch(switches::kInitialURL)) {
-    std::string url_switch =
-        command_line->GetSwitchValueASCII(switches::kInitialURL);
+  bool has_initial_url_switch = command_line->HasSwitch(switches::kInitialURL);
+
+  const base::CommandLine::StringVector& args = command_line->GetArgs();
+  if (has_initial_url_switch || !args.empty()) {
+    std::string url_string =
+        has_initial_url_switch
+            ? command_line->GetSwitchValueASCII(switches::kInitialURL)
+        : args.empty() ? std::string()
```

Comment:
If has_initial_url_switch is false, then `args.empty()` must evaluate to true for us to be in this conditional, so this can be removed

Reply:
Done.